docs/tutorials/misc/300_otcp.ipynb

```diff
@@ -0,0 +1,780 @@
+{
```

Comment:
<p>Same, I'd remove this.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/613/discussion/#comment-2023228930'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>